client/.eslintrc.json

```diff
@@ -5,7 +5,7 @@
     "airbnb-typescript"
   ],
   "parserOptions": {
-    "project": "./tsconfig.json"
+    "project": "./client/tsconfig.json"
```

Comment:
Can you change this back to just be `./tsconfig.json`? The build process on Netlify needs a path relative to the `./client` folder. 

If you're dealing with the errors from ESLint then you could try setting the tsconfigRootDir as they do here: https://stackoverflow.com/questions/64933543/parsing-error-cannot-read-file-tsconfig-json-eslint

Or you could just open the `client` folder in VSCode rather than the whole folder.